$NetBSD: patch-ac,v 1.6 2017/03/30 18:44:13 nat Exp $

Unconditionally include errno.h since extern int errno does not
conform to POSIX in general.

--- timidity/timidity.h.orig	2012-06-01 03:40:44.000000000 +0000
+++ timidity/timidity.h
@@ -451,10 +451,7 @@ typedef double FLOAT_T;
 
 #include <stdio.h>
 
-#ifdef HAVE_ERRNO_H
 #include <errno.h>
-#endif /* HAVE_ERRNO_H */
-extern int errno;
 
 #ifdef HAVE_MACHINE_ENDIAN_H
 #include <machine/endian.h> /* for __byte_swap_*() */
